src/vulkan/transfer_image.cc

```diff
@@ -203,12 +219,17 @@ VkBufferImageCopy TransferImage::CreateBufferImageCopy(uint32_t mip_level) {
 }
 
 void TransferImage::CopyToHost(CommandBuffer* command_buffer) {
+  const VkImageAspectFlagBits aspects[] = {VK_IMAGE_ASPECT_COLOR_BIT,
+                                           VK_IMAGE_ASPECT_DEPTH_BIT,
+                                           VK_IMAGE_ASPECT_STENCIL_BIT};
   std::vector<VkBufferImageCopy> copy_regions;
   uint32_t last_mip_level = used_mip_levels_ == VK_REMAINING_MIP_LEVELS
                                 ? mip_levels_
                                 : base_mip_level_ + used_mip_levels_;
   for (uint32_t i = base_mip_level_; i < last_mip_level; i++)
-    copy_regions.push_back(CreateBufferImageCopy(i));
+    for (auto aspect : aspects)
```

Comment:
nit: Both these for()'s should have {}'s around ther bodies as they're more then 1 line long.